web/src/routes/SearchPage.tsx

```diff
@@ -100,16 +100,17 @@ const SearchPage = ({ city, cityCode, languageCode, pathname }: CityRouteProps):
       <Helmet pageTitle={pageTitle} languageChangePaths={languageChangePaths} cityModel={city} />
       {SearchBar}
       <List>
-        {results.map(({ title, content, path, thumbnail }) => (
-          <SearchListItem
-            title={title}
-            contentWithoutHtml={parseHTML(content)}
-            key={path}
-            query={query}
-            path={path}
-            thumbnail={thumbnail}
-          />
-        ))}
+        {query.length > 0 &&
```

Comment:
:wrench: Could we move this up and only render the `List` conditionally as well? Or does that lead to problems?